package.json

```diff
@@ -57,6 +57,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
     "ember-ajax": "^2.5.6",
+    "ember-app-scheduler": "0.0.5",
```

Comment:
@briangonzalez I think this should be a prod dependency, otherwise this will fail unless all users also explicitly add this dep